examples/js_binary/BUILD.bazel

```diff
@@ -95,6 +95,28 @@ diff_test(
     file2 = "out2_local",
 )
 
+# Also test with run_from_execroot execution
+js_run_binary(
+    name = "run2_run_from_execroot",
+    srcs = [],
+    outs = ["out2_run_from_execroot"],
+    args = ["out2_run_from_execroot"],
+    chdir = package_name(),
+    # Request that the rules_js launcher prints extra information
+    log_level = "debug",
+    run_from_execroot = True,
```

Comment:
what's the downside if this defaults to True?

I barely understand the difference between the two ways of running this, so I'd have a hard time explaining to users. I'm concerned most people won't discover that their problem can be fixed with this flag...

Reply:
yeah; that should work as a better default. flipped it and everything is green. 🌮